src/test/java/com/cruru/process/controller/ProcessControllerTest.java

```diff
@@ -29,13 +29,17 @@ class ProcessControllerTest {
 
     private Dashboard dashboard;
 
-    private Process process;
+    private Process process1;
+    private Process process2;
+    private Process process3;
 
     @BeforeEach
     void setUp() {
         RestAssured.port = port;
         dashboard = dashboardRepository.save(new Dashboard("name", null));
-        process = processRepository.save(new Process(0, "면접", "1차 면접", dashboard));
+        process1 = processRepository.save(new Process(0, "면접", "1차 면접", dashboard));
+        process2 = processRepository.save(new Process(1, "면접", "1차 면접", dashboard));
+        process3 = processRepository.save(new Process(2, "면접", "1차 면접", dashboard));
```

Comment:
```suggestion
        processRepository.saveAll(
            List.of(
                new Process(0, "면접", "1차 면접", dashboard),
                new Process(1, "면접", "1차 면접", dashboard),
                new Process(2, "면접", "1차 면접", dashboard)
                )
        );
```

전 save가 세 개 이상부터 saveAll()을 사용하는 편이에요. saveAll 사용을 고려해보세요:)